frontend/src/metabase/components/Toaster/Toaster.styled.tsx

```diff
@@ -54,3 +53,11 @@ export const ToasterButton = styled.button`
     background-color: ${alpha(color("bg-white"), 0.3)};
   }
 `;
+
+export const ToasterDismiss = styled.button`
+  cursor: pointer;
+  transition: color 200ms ease;
```

Comment:
You could add `color: ${color("bg-dark")};` here and remove `.Icon-close` from the next line with `color={color("bg-dark")}` prop on the `Icon` component